packages/vaadin-combo-box/src/vaadin-combo-box-dropdown-wrapper.js

```diff
@@ -49,6 +49,12 @@ class ComboBoxDropdownWrapperElement extends PolymerElement {
               /* Fixes scrollbar disappearing when 'Show scroll bars: Always' enabled in Safari */
               box-shadow: 0 0 0 white;
             }
+
+            #selector {
+              border-width: var(--_vaadin-combo-box-items-container-border-width);
+              border-style: var(--_vaadin-combo-box-items-container-border-style);
+              border-color: var(--_vaadin-combo-box-items-container-border-color);
+            }
```

Comment:
I wonder if this really needed three custom properties. The main purpose for this has been to provide padding/white space around the list.

Reply:
The props are private so no harm done, can be optimized. I first tried with one: `...items-container-border`, but MD has `8px 0` for the width so it didn't work.